scripts/rpc.py

```diff
@@ -1702,6 +1703,7 @@ def nvmf_create_transport(args):
     p.add_argument('-o', '--c2h-success', action='store_false', help='Disable C2H success optimization. Relevant only for TCP transport')
     p.add_argument('-f', '--dif-insert-or-strip', action='store_true', help='Enable DIF insert/strip. Relevant only for TCP transport')
     p.add_argument('-y', '--sock-priority', help='The sock priority of the tcp connection. Relevant only for TCP transport', type=int)
+    p.add_argument('-b', '--wr-batching', action='store_true', help='Disable work requests batching. Relevant only for RDMA transport')
```

Comment:
Description says that this flag disables batching. But it stores True when specified and will send 'true' to SPDK.
So, looks like it enables batching and there is no way to disable it via RPC.
Please check how it is done for bdev_nvme_set_options --disable-delay-cmd-submit parameter.